modules/nw-egress-ips-about.adoc

```diff
@@ -1,7 +1,108 @@
 // Module included in the following assemblies:
 //
-// * networking/assigning-egress-ips.adoc
+// * networking/openshift_sdn/assigning-egress-ips.adoc
+// * networking/ovn_kubernetes_network_provider/assigning-egress-ips.adoc
 
+ifeval::["{context}" == "egress-ips"]
+:openshift-sdn:
+endif::[]
+ifeval::["{context}" == "ovn-kubernetes-configuring-egress-ips"]
+:ovn:
+endif::[]
+
+ifdef::ovn[]
+[id="nw-egress-ips-about_{context}"]
+= Egress IP address architectural design and implementation
+
+The {product-title} egress IP address functionality allows you to ensure that the traffic from one or more pods in a namespace have a consistent source IP address for services outside the cluster network.
+
+For example, you might have a pod that periodically queries a database hosted on a server outside of your cluster. To enforce access requirements for the server, a packet filtering device is configured to allow traffic only from specific IP addresses.
+You can configure an specific external IP address for the pod making the requests to the server, to ensure that you can reliably allow access to the server from only that specific pod.
+
+An Egress IP address is implemented as additional IP address on the primary network interface of the node and must be in the same subnet as the primary IP address of the node.
+
+[id="nw-egress-ips-considerations_{context}"]
+== Assignment of egress IPs to pods
+
+To assign one or more egress IPs to a namespace or specific pods in a namespace, the following conditions must be satisfied:
+
+- You must have at least one node in your cluster with the `k8s.ovn.org/egress-assignable: ""` label.
+- You must create an `EgressIP` object specifying one or more egress IPs to use as the source IP address for traffic leaving the cluster from pods in a project.
+
+[IMPORTANT]
+====
+If you create `EgressIP` objects prior to labeling any nodes in your cluster for egress IP assignment, {product-title} might assign every egress IP address to the first node with the `k8s.ovn.org/egress-assignable: ""` label.
+
+To avoid this, always apply the label to the nodes you intent to host egress IP addresses prior to creating any `EgressIP` objects.
+====
+
+[id="nw-egress-ips-node-assignment_{context}"]
+== Assignment of egress IPs to nodes
+
+When creating an `EgressIP` object, the following conditions apply to nodes labeled with `k8s.ovn.org/egress-assignable: ""`:
+
+- An egress IP address is never assigned to more than one node at a time.
+- Egress IP addresses are equally balanced between available nodes.
```

Comment:
`Egress IP addresses are equally balanced between available nodes.` ->  `Egress IP addresses are equally balanced between available nodes that can host the egress IP.`

Reply:
I am saying this because even if a node has the label `k8s.ovn.org/egress-assignable: ""` it might not be able to host it, if the egress IP is not on its subnet.